src/alloy/README.md

```diff
@@ -0,0 +1,37 @@
+
+Three tools in the space of log collection were considered:
+1. [Vector](https://vector.dev/): Opensource and maintained by Datadog, Vector provides input integrations with Kubernetes logs, arbitrary files, and [other sources](https://vector.dev/docs/reference/configuration/sources/). It has the necessary export integrations with Loki, S3, and a [number of other targets](https://vector.dev/docs/reference/configuration/sinks/). Vector is a newer tool that has not yet reached a 1.0 release, but has risen in popularity due to its performance improvements over other tools.
+1. [FluentBit](https://fluentbit.io/): Fluentbit was historically used in Big Bang and supports file based inputs as well as [other targets](https://docs.fluentbit.io/manual/pipeline/inputs). It also supports the necessary export integrations (Loki, S3, OpenTelemetry and [others](https://docs.fluentbit.io/manual/pipeline/outputs)). FluentBit is a CNCF graduated project and is relatively mature.
+1. [Grafana Alloy](https://grafana.com/docs/alloy/latest/): Alloy is a distribution of the OpenTelemetry Collector, opensource and maintained by Grafana Labs. It supports the necessary [inputs and outputs](https://grafana.com/docs/alloy/latest/reference/components/) (local file/k8s logs, Loki and S3). As a distribution of OTel it supports vendor-agnostic logging formats and can be integrated with numerous other tools through the OTel ecosystem. While Alloy itself is relatively new, it is built on the previous codebase of Grafana Agent and the existing OTel framework.
+
+### Decision and Impact
+
+Grafana Alloy has been chosen as our replacement for Promtail. Primary motivations include:
+1. It is positioned as the "successor"/more feature-rich alternative to Promtail, to include [migration documentation](https://grafana.com/docs/alloy/latest/set-up/migrate/from-promtail/) focused on the switch. This makes it very easy to switch from Promtail to Alloy with little end user impact.
+1. As Alloy is part of the Grafana ecosystem it has good integrations with Loki to provide the enriched log data we have come to expect from Promtail (simple approach to labelling logs, etc).
+1. Through Alloy's S3 integration we can export logs to an additional storage location in generic formats (raw logs, OTel JSON) without any modification - no need to provide Alloy with edit/delete permissions.
+1. By choosing a distribution of the OTel Collector we have flexibility in the future to switch to a different distribution of the OTel collector and/or easily swap out our logging backend (Loki) for something else that fits in the OTel Framework without needing to ensure specific tool compatibility. OTel is part of the CNCF (Incubating) and seems to be on a good trajectory.
```

Comment:
Not as it stands today your recommendation would be continue to rely on alloy shipping logs to Loki in the Loki native format?  I don't think it is a huge sticking point but curious with Loki 3.0 supporting otel native format we _could_ use that but it's probably an inferior experience at this point?

Reply:
Yeah I think that is something we could look at and evaluate as a follow on here - don't think it's worth pursuing as part of this initial switch over. A few considerations here:
- We aren't on Loki 3.0 yet, although there is a pending PR for that update. One of the main challenges in that upgrade is the schema change which is required for OTel support.
- (unsure, question) would switching to OTel actually change the way Loki stores logs? I think this would be the main/only real benefit - if it could enable us to switch to a different backend easier in the future.
- Would we lose anything by switching (similar to your note of "inferior experience") - might make more sense just to stay with the native integration if its an option.